### PR TITLE
doc: add container kill example in pod chaos experiments doc.

### LINKED ISSUE #950: Document: add document about kill container
## Problem

We lack the document about `Container Kill` in [Pod Chaos](https://chaos-mesh.org/docs/user_guides/podchaos_experiment).

## Description

Add the description of `Container Kill` in [Pod Chaos](https://chaos-mesh.org/docs/user_guides/podchaos_experiment).

## Implementation

Include the sample configuration file of `Container Kill` and some explain, just like `Pod Failure` and `Pod Kill`.

Refer to [podchaos_types.go](https://github.com/chaos-mesh/chaos-mesh/blob/master/api/v1alpha1/podchaos_types.go) for more details about `Container Kill`.

### PR DESCRIPTION
**Pull request by Colstuwjx**
### What problem does this PR solve?

Fixed #950 .

### What is changed and how does it work?
Revised the document.

### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Breaking backward compatibility

Related changes

- [ ] Need to update the documentation

### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: website/docs/chaos_experiments/pod_chaos.md [hunk 1]
```diff
@@ -71,4 +66,38 @@ spec:
...
+  selector:
+    labelSelectors:
+      "app.kubernetes.io/component": "monitor"
+  scheduler:
+    cron: "@every 30s"
+```
+
+The detailed description of each field in the configuration template see [`field description`](#fields-description).
+
+## fields description
```
Comment (dcalvin) [inline website/docs/chaos_experiments/pod_chaos.md:94]:
```suggestion
## Fields description
```

File: website/docs/chaos_experiments/pod_chaos.md [hunk 2]
```diff
@@ -71,4 +66,38 @@ spec:
...
+  mode: one
+  containerName: "prometheus"
+  selector:
+    labelSelectors:
+      "app.kubernetes.io/component": "monitor"
+  scheduler:
+    cron: "@every 30s"
+```
+
+The detailed description of each field in the configuration template see [`field description`](#fields-description).
```
Comment (dcalvin) [inline website/docs/chaos_experiments/pod_chaos.md:92]:
```suggestion
For a detailed description of each field in the configuration template, see [`Field description`](#fields-description).
```

**Review by dcalvin — APPROVED** [review]
LGTM

**Review by cwen0 — APPROVED** [review]
LGTM

**Comment by Colstuwjx** [comment]
Why the bot merge failed? @cwen0

**Comment by cwen0** [comment]
@Colstuwjx CI broke due to failing to download helm!

**Pull request merged**